theme_graphene/data/theme_website_design.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="theme_graphene.design" model="theme.website.design">
+    <field name="layout">'boxed'</field>
+    <field name="theme_id" ref="base.module_theme_graphene"/>
```

Comment:
You are actually filling the "theme.utils" many2one with a "ir.module.module" record, funny if it works ^^ Anyways, like I said in the other PR: you don't need that. You are in theme_graphene, you should never have to say "I am a theme_graphene record".